lib/eapol_test.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "erb"
+
+class EapolTest
+  TEMPLATE_PATH = File.dirname(__FILE__) + "/peap-mschapv2.conf.erb"
+
+  def self.create(*args)
```

Comment:
I found this a little confusing. This method returns whatever you do in the yield block and not the ```EapolTest``` which is what I would expect.

Reply:
possibly needs renamed... it's creating a test, not (specifically) an instance.... `create_test` maybe.

Reply:
Renamed to `make_test`